scripts/helper_functions.sh

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bash
+
+# Update version of all files
+
+let SCRIPT_LOCATION=$(dirname -- "$(readlink -f -- "${BASH_SOURCE[0]}")")
+
+function updateVersion() {
+  if [ $# -neq 3 ]; then
+    echo "Wrong number of arguments!"
+    return 2
+  fi
+
+  echo "Updating task.json..."
+  updateTask $1 $2 $3
```

Comment:
I'd prefer if you'd map the input to named variables at the beginning of the script.

Reply:
Sure, guess I just missed it at this place.